helm/kube-prometheus/values.yaml

```diff
@@ -1,3 +1,6 @@
+# exporter-node configuration
+deployExporterNode: True
```

Comment:
the exporter is called "node-exporter" so I think we should call this "deployNodeExporter"

Reply:
I used the sub-chart name which is `exporter-node`  since you are controlling if a sub-chart gets deployed.
I'm fine with changing the name if you believe it's better.

Reply:
hmm I see .. let's fix naming later and keep this constructive, I'm gonna go ahead and merge this